packages/mgt-spfx/.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Is this launch config file used? It doesn't look like this package can run on it's own.

Reply:
Good catch. Let me remove it